src/Infrastructure/Services/BookingService.cs

```diff
@@ -49,8 +52,8 @@ public async Task<bool> RemoveAsync(Guid Id)
 
         public async Task<IEnumerable<GetBookingDTO>> SearchByUserIdAsync(Guid UserId)
         {
-            var bookings = await _unitOfWork.Bookings.Search(c => c.UserId.Contains(UserId));
-            if (bookings = null)
+            var bookings = _unitOfWork.Bookings.Search(c => c.UserId.Equals(UserId), false);
```

Comment:
As I recommended before, add default value for tracking param in interface's method

Reply:
created befault values in repositoru methods